frontend/util/sort.js

```diff
@@ -0,0 +1,13 @@
+export const sortArrayOfObjects = (wifi_networks, key) => {
```

Comment:
I am not sure this should be a general util function, as
1. It has special handling for the `Default` sorting
2. it always uses numeric sorting, which might be quite unexpected 
Either change the name, or put it into `Wifi.jsx`